packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -704,7 +704,7 @@ describe('<PersonalDetails/>', () => {
 
         expect(screen.getByText(tax_residence_pop_over_text)).toBeInTheDocument();
 
-        fireEvent.scroll(screen.getByRole('heading', { name: /account opening reason/i }), {
+        fireEvent.scroll(screen.getByTestId('personal-details-container'), {
```

Comment:
```suggestion
        fireEvent.scroll(screen.getByTestId('dt_personal_details_container'), {
```
As far as I know, we have a convention to name the test ids in snake_case starting with dt_.